design/cli-install-changes.md

```diff
@@ -0,0 +1,426 @@
+        --default-backup-ttl duration                how long to wait by default before backups can be garbage collected (default 720h0m0s)
+        --disable-controllers strings                list of controllers to disable on startup. Valid values are backup,backup-sync,schedule,gc,backup-deletion,restore,download-request,restic-repo,server-status-request
+        --log-format                                 the format for log output. Valid values are text, json. (default text)
+        --log-level                                  the level at which to log. Valid values are debug, info, warning, error, fatal, panic. (default info)
+        --metrics-address string                     the address to expose prometheus metrics (default ":8085")
+        --plugin-dir string                          directory containing Velero plugins (default "/plugins")
+        --profiler-address string                    the address to expose the pprof profiler (default "localhost:6060")
+        --restore-only                               run in a mode where only restores are allowed; backups, schedules, and garbage-collection are all disabled. DEPRECATED: this flag will be removed in v2.0. Use read-only backup storage locations instead.
+        --restore-resource-priorities strings        desired order of resource restores; any resource not in the list will be restored alphabetically after the prioritized resources (default [namespaces,storageclasses,persistentvolumes,persistentvolumeclaims,secrets,configmaps,serviceaccounts,limitranges,pods,replicaset,customresourcedefinitions])
+        --terminating-resource-timeout duration      how long to wait on persistent volumes and namespaces to terminate during a restore before timing out (default 10m0s)
+
+      restic                                        Configuration for restic operations.
+        --default-prune-frequency duration          how often 'restic prune' is run for restic repositories by default. Optional.
+        --pod-annotations mapStringString           annotations to add to the Velero and restic pods. Optional. Format is key1=value1,key2=value2
+        --pod-cpu-limit string                      CPU limit for restic pod. A value of "0" is treated as unbounded. Optional. (default "0")
+        --pod-cpu-request string                    CPU request for restic pod. A value of "0" is treated as unbounded. Optional. (default "0")
+        --pod-mem-limit string                      memory limit for restic pod. A value of "0" is treated as unbounded. Optional. (default "0")
+        --pod-mem-request string                    memory request for restic pod. A value of "0" is treated as unbounded. Optional. (default "0")
+        --deployment                                create restic deployment. Default is false. Optional. Other flags will only work if set to true. (NEW, was `velero install use-restic`)
```

Comment:
I would think that if the user doesn't want restic installed, they just won't run `velero config restic`. So I think this `--deployment` flag can probably be dropped?

Reply:
Yes, that makes sense.